nginx/microcache.conf

```diff
@@ -0,0 +1,8 @@
+# vim: syntax=nginx ts=4 sw=4 sts=4 sr noet
```

Comment:
this is an NGINX microcache of 1s which can help reduce the load from having multiple edge servers hit the backend for the same endpoint at once. It's probably not necessary, but having it gives me a little peace of mind.